src/components/project-pages/KaiLandre.tsx

```diff
@@ -1,33 +1,28 @@
 import React from "react";
 import styled from "styled-components";
-import Timer from "../Timer";
 import { FlexboxProps, flexbox, layout, LayoutProps } from "styled-system";
-import dragon from "../assets/project-page/dragon.png";
+import KaiLandrePreview from "./KaiLandrePreview";
+import ReactPlayer from 'react-player/lazy'
 
 const Main = styled.main<FlexboxProps>`
   display: flex;
-  height: 100vh;
-  overflow: hidden;
-  flex-direction: column;
+  height: 100%;
   ${flexbox}
 `;
 
-const Container = styled.div<FlexboxProps>`
-  display: flex;
-  ${flexbox};
-`;
-
-const Img = styled.img<LayoutProps>`
+const Container = styled.div<LayoutProps>`
   ${layout};
 `;
 
 const KaiLandre = () => {
+  const url = 'https://www.youtube.com/watch?v=AGSH_acR4wA'
```

Comment:
is it definitely a youtube video that you'll be sent? and not Vimeo or something else?

Reply:
He said YouTube but [the React player also works for Vimeo](https://cookpete.com/react-player/)